### PR TITLE
Use gradlew in native image docs

### PR DESCRIPTION
**Pull request by klemen-df**
Updated version, updated sinstructions

### CODE REVIEW

File: build-parent/pom.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
         <!-- These properties are needed in order for them to be resolvable by the documentation -->
         <!-- The Graal version we suggest using in documentation - as that's
            what we work with by self downloading it: -->
-        <graal-sdk.version-for-documentation>22.0.0.2</graal-sdk.version-for-documentation>
+        <graal-sdk.version-for-documentation>22.0</graal-sdk.version-for-documentation>
```
Comment (glefloch) [inline build-parent/pom.xml:42]: Is that change necessary for documentation update?

File: docs/src/main/asciidoc/building-native-image.adoc [hunk 2]
```diff
@@ -566,6 +566,8 @@ Sample Dockerfile for building with Gradle:
 ----
 ## Stage 1 : build with maven builder image with native capabilities
 FROM quay.io/quarkus/ubi-quarkus-native-image:{graalvm-flavor} AS build
+USER root
+RUN microdnf install findutils
```
Comment (glefloch) [inline docs/src/main/asciidoc/building-native-image.adoc:570]: is find utils necessary for building the native artifact?

File: docs/pom.xml [hunk 3]
```diff
@@ -2786,7 +2786,7 @@
                         <quarkus-version>${project.version}</quarkus-version>
                         <surefire-version>${version.surefire.plugin}</surefire-version>
                         <graalvm-version>${graal-sdk.version-for-documentation}</graalvm-version>
-                        <graalvm-flavor>${graal-sdk.version-for-documentation}-java11</graalvm-flavor>
+                        <graalvm-flavor>${graal-sdk.version-for-documentation}-java17</graalvm-flavor>
```
Comment (glefloch) [inline docs/pom.xml:2789]: I know if we want to set all documentation reference to java 17 as quarkus is still based on Java 11

**Review by glefloch — COMMENTED** [review]
Thanks for the PR, I left few comments.

File: docs/src/main/asciidoc/building-native-image.adoc (hunk 2 above)
Comment (KlemenDanfoss) [reply docs/src/main/asciidoc/building-native-image.adoc:570]: gradlew ( https://github.com/gradle/gradle/blob/master/gradlew , line 235 ) uses xargs which are not in the image by default.

**Comment by geoand** [comment]
Just a note that we'll need to have the commits squashed once everything else is addressed

File: docs/pom.xml (hunk 3 above)
Comment (KlemenDanfoss) [reply docs/pom.xml]: Reverted.

File: build-parent/pom.xml (hunk 1 above)
Comment (KlemenDanfoss) [reply build-parent/pom.xml]: Reverted.

**Review by glefloch — COMMENTED** [review]
LGTM, @KlemenDanfoss could you squash your commits and name it with the intent of the modification ?

**Comment by KlemenDanfoss** [comment]
Done.

**Review by glefloch — COMMENTED** [review]
Thanks for those fix, the overall looks good to me. Could you just rephrase your commit message to something like "Use gradlew in native image docs", this will be used for the changelog.

**Review by glefloch — APPROVED** [review]
